exist-core/src/main/java/org/exist/collections/CollectionConfiguration.java

```diff
@@ -90,6 +90,8 @@
 
     private final BrokerPool pool;
 
+    private boolean skipTriggers = false;
```

Comment:
This isn't a suitable stable place to keep this flag!

Each time the `collection.xconf` is changed your setting will be lost, as this class is GC'd and a new instance created from the `collection.xconf` file.

Reply:
Ok, where would you place that flag? Obviously some flag would have to be kept somewhere

Reply:
I don't have a suggestion, I just noticed that where you have it won't work as you intended.

Reply:
Thanks for the heads-up